pkg/validation/options.go

```diff
@@ -411,3 +417,71 @@ func parseURL(toParse string, urltype string, msgs []string) (*url.URL, []string
 	}
 	return parsed, msgs
 }
+
+// CallbackErrorRedirect - Allow automated redirection to occur within the AuthCallback when an error occurs
+type CallbackErrorRedirect struct {
+	ErrorString string
+	Pattern     string
+	RedirectURL string
+}
```

Comment:
This should be in the options package really, I'd prefer this to be embedded in the options then constructed in the validation package.

If you take a look at the legacy options and how they're structured, we have some conversion there. I think we should make this "legacy" and have the conversion done as part of the legacy `ToOptions`, WDYT?

Reply:
Bump